platform/linux-generic/odp_stash.c

```diff
@@ -116,9 +180,9 @@ int odp_stash_capability(odp_stash_capability_t *capa, odp_stash_type_t type)
 	(void)type;
 	memset(capa, 0, sizeof(odp_stash_capability_t));
 
-	capa->max_stashes_any_type = CONFIG_MAX_STASHES - CONFIG_INTERNAL_STASHES;
-	capa->max_stashes          = CONFIG_MAX_STASHES - CONFIG_INTERNAL_STASHES;
-	capa->max_num_obj          = MAX_RING_SIZE;
+	capa->max_stashes_any_type = stash_global->max_num - stash_global->num_internal;
+	capa->max_stashes          = stash_global->max_num - stash_global->num_internal;
```

Comment:
Not compile-time anymore, so `stash_global->max_num - stash_global->num_internal` could be calculated just once.

Reply:
Fixed in V2.